api/utils/run_cache.ts

```diff
@@ -17,16 +16,6 @@ export const decodePersistedTableMetadata = (protoString: string) => {
   return dataform.PersistedTableMetadata.decode(encodedProto);
 };
 
-export const encodePersistedTableMetadata = (table: dataform.PersistedTableMetadata) => {
-  const encodedProtoBuffer = Buffer.from(dataform.PersistedTableMetadata.encode(table).finish());
-  return encodedProtoBuffer.toString("base64");
-};
-
-export const buildQuery = (targetName: string, table: dataform.PersistedTableMetadata) => {
-  const encodedProtoString = encodePersistedTableMetadata(table);
-  const query = `SELECT 
-          '${targetName}' AS target_name,
-          '${JSON.stringify(table.toJSON())}' AS metadata_json,
-          '${encodedProtoString}' as metadata_proto`;
-  return query;
+export const encodePersistedTableMetadata = (table: dataform.IPersistedTableMetadata) => {
+  return Buffer.from(dataform.PersistedTableMetadata.encode(table).finish()).toString("base64");
```

Comment:
r.e. what you said, is it worth us just writing a quick test somewhere to check that protobuf.js is at least consistent with itself on how it encodes protos? e.g. if we give it plain ojects with differently ordered keys, that this method produces the same buffer.

I'm worried now that we refactor some code and invalidate everyone's cache

Reply:
Also feels like this should just have a unit test case in general

Reply:
but we don't care whether a different encoding happens for `PersistedTableMetadata` - it's not used as a key, nor is it intended to be.